include/seqan3/core/type_traits/transformation_trait_or.hpp

```diff
@@ -21,6 +21,12 @@
 
 namespace seqan3::detail
 {
+template <typename t>
+inline constexpr bool is_trait_v{false};
+
+template <typename t>
+    requires requires { typename t::type; }
+inline constexpr bool is_trait_v<t>{true};
```

Comment:
Needs documentation

Reply:
@h-2 This was something you added for me! Any suggestions on documentation?